src/LanguageServer/Protocol.TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -390,13 +382,19 @@ internal EditorTestWorkspace CreateWorkspace(
         /// Waits for the async operations on the workspace to complete.
         /// This ensures that events like workspace registration / workspace changes are processed by the time we exit this method.
         /// </summary>
-        protected static async Task WaitForWorkspaceOperationsAsync(EditorTestWorkspace workspace)
+        protected static async Task WaitForWorkspaceOperationsAsync<TDocument, TProject, TSolution>(TestWorkspace<TDocument, TProject, TSolution> workspace)
```

Comment:
generics are required so this can be passed an EditorTestWorkspace (for a few tests that use that) or LspTestWorkspace (almost everything else).